### PR TITLE
Block scaling when camera model is fisheye

### PR DESCRIPTION
**Pull request by DavoliLuca**
Closes: SW-64240

### CODE REVIEW

File: camera_calibration/src/camera_calibration/camera_calibrator.py [hunk 1]
```diff
@@ -364,7 +364,11 @@ def on_model_change(self, model_select_val):
 
     def on_scale(self, scalevalue):
         if self.c.calibrated:
-            self.c.set_alpha(scalevalue / 100.0)
+            if self.c.camera_model is not CAMERA_MODEL.FISHEYE:
```
Comment (g-gemignani) [inline camera_calibration/src/camera_calibration/camera_calibrator.py]: Nit: you could merge the two ifs together. I believe it's better for readability.

Comment (DavoliLuca) [reply camera_calibration/src/camera_calibration/camera_calibrator.py]: changed

**Review by brinij — APPROVED** [review]

File: camera_calibration/src/camera_calibration/camera_calibrator.py [hunk 2]
```diff
@@ -363,8 +363,11 @@ def on_model_change(self, model_select_val):
         self.c.set_cammodel( CAMERA_MODEL.PINHOLE if model_select_val < 0.5 else CAMERA_MODEL.FISHEYE)
 
     def on_scale(self, scalevalue):
-        if self.c.calibrated:
+        if self.c.calibrated and self.c.camera_model is not CAMERA_MODEL.FISHEYE:
             self.c.set_alpha(scalevalue / 100.0)
+        else:
```
Comment (g-gemignani) [inline camera_calibration/src/camera_calibration/camera_calibrator.py]: By addressing my comment, you changed the logic. Beforehand, we were not doing anything if not self.c.calibrated. Now we are warning and setting scalevalue. Is it correct?

File: camera_calibration/src/camera_calibration/camera_calibrator.py [hunk 3]
```diff
@@ -363,8 +363,11 @@ def on_model_change(self, model_select_val):
         self.c.set_cammodel( CAMERA_MODEL.PINHOLE if model_select_val < 0.5 else CAMERA_MODEL.FISHEYE)
 
     def on_scale(self, scalevalue):
-        if self.c.calibrated:
+        if self.c.calibrated and self.c.camera_model is not CAMERA_MODEL.FISHEYE:
             self.c.set_alpha(scalevalue / 100.0)
+        elif self.c.calibrated and self.c.camera_model is CAMERA_MODEL.FISHEYE:
```
Comment (g-gemignani) [inline camera_calibration/src/camera_calibration/camera_calibrator.py]: you do not need the and part. Remove it

**Pull request merged**